stan-dev/math
# Renaming probability distribution functions

#### Submisison Checklist
#### Summary:

Renames the probability distribution functions to match the language definitions. 
#### Intended Effect:
- Renamed `_log` functions to `_lpdf` or `_lpmf` depending on the type of the first argument
- Renamed `_cdf_log` to `_lcdf`
- Renamed ``ccdf_log` to `_lccdf`
- Kept the `_log` functions and had it delegate to the appropriate `_lpdf`, `_lpmf`, `_lcdf`, or `_lccdf` function.
- Added doxygen doc to the `_log` function indicating that the function is deprecated.
#### How to Verify:

Look at the diff. Here are the things we should be careful about:
- check to see the header guard is correct
- check the error message
- check the delegation
#### Side Effects:

None. We've left the old behavior intact.
#### Documentation:

Included in C++.
#### Reviewer Suggestions:

Anyone.
#### Copyright and Licensing

Please list the copyright holder for the work you are submitting (this will be you or your assignee, such as a university or company):

Columbia University

By submitting this pull request, the copyright holder is agreeing to license the submitted work under the following licenses:
- Code: BSD 3-clause (https://opensource.org/licenses/BSD-3-Clause)
- Documentation: CC-BY 4.0 (https://creativecommons.org/licenses/by/4.0/)
